src/Compilers/CSharp/Portable/Binder/LocalScopeBinder.cs

```diff
@@ -230,74 +215,6 @@ protected ImmutableArray<LocalSymbol> BuildLocals(SyntaxList<StatementSyntax> st
 
             return locals.ToImmutableAndFree();
         }
-
-        internal void CollectLocalsFromDeconstruction(
```

Comment:
`CollectLocalsFromDeconstruction` was moved to `ForEachLoopBinder` since it is now the only remaining consumer.